rerun_py/docs/gen_common_index.py

```diff
@@ -54,88 +54,58 @@ class Section:
         class_list=[],
     ),
     Section(
-        title="Viewer Control",
+        title="Logging",
         module_summary=None,
-        func_list=["save"],
-        class_list=[],
-    ),
-    Section(
-        title="Time",
-        module_summary=None,
-        func_list=["set_time_sequence", "set_time_seconds", "set_time_nanos"],
+        func_list=["log", "set_time_sequence", "set_time_seconds", "set_time_nanos"],
         class_list=[],
     ),
     Section(
         title="Spatial Primitives",
         module_summary=None,
-        func_list=[
-            "log_point",
-            "log_points",
-            "log_rect",
-            "log_rects",
-            "log_obb",
-            "log_line_strip",
-            "log_line_segments",
-            "log_arrow",
-            "log_mesh",
-            "log_meshes",
-            "log_mesh_file",
-        ],
-        class_list=[],
+        func_list=[],
+        class_list=["Points2D", "Points3D", "Boxes2D", "Boxes3D", "LineStrips2D", "LineStrips3D", "Mesh3D", "Asset3D"],
```

Comment:
Arrow is missing

Reply:
Also: what is this order? O.o

Reply:
Ordered by increasing complexity of object. Same as how the functions were ordered.